### PR TITLE
Fix EternusDX support for Liberty+

### PR DESCRIPTION
**Pull request by dirkmueller**
pool was renamed to snappool and the configuration file parameter
was renamed as well

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/cinder_eternus_dx_config.xml.erb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <EternusUser><%= @eternus_params["user"] %></EternusUser>
 <EternusPassword><%= @eternus_params["password"] %></EternusPassword>
 <EternusPool><%= @eternus_params["pool"] %></EternusPool>
```
Comment (toabctl) [inline chef/cookbooks/cinder/templates/default/cinder_eternus_dx_config.xml.erb]: According to the commit message, "pool was renamed to snappool.." so please remove ```<EternusPool></EternusPool>```

File: chef/data_bags/crowbar/template-cinder.json [hunk 2]
```diff
@@ -70,6 +70,7 @@
           "user": "",
           "password": "",
           "pool": "",
```
Comment (toabctl) [inline chef/data_bags/crowbar/template-cinder.json]: ```pool``` should be removed.

File: chef/data_bags/crowbar/template-cinder.schema [hunk 3]
```diff
@@ -196,6 +197,7 @@
                       "user": { "type": "str", "required": true },
                       "password": { "type": "str", "required": true },
                       "pool": { "type": "str", "required": true },
```
Comment (toabctl) [inline chef/data_bags/crowbar/template-cinder.schema]: ```pool``` should be removed

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml [hunk 4]
```diff
@@ -126,6 +126,7 @@
               = string_field %w(volumes {{@index}} eternus user)
               = password_field %w(volumes {{@index}} eternus password)
               = string_field %w(volumes {{@index}} eternus pool)
```
Comment (toabctl) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: ```pool``` should be removed

File: chef/cookbooks/cinder/templates/default/cinder_eternus_dx_config.xml.erb (hunk 1 above)
Comment (dirkmueller) [inline chef/cookbooks/cinder/templates/default/cinder_eternus_dx_config.xml.erb]: no, eternuspool still exists, it is the storage pool now (it was the snapshot pool before)

File: chef/data_bags/crowbar/template-cinder.json (hunk 2 above)
Comment (dirkmueller) [inline chef/data_bags/crowbar/template-cinder.json]: no

File: chef/data_bags/crowbar/template-cinder.schema (hunk 3 above)
Comment (dirkmueller) [inline chef/data_bags/crowbar/template-cinder.schema]: no

File: crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml (hunk 4 above)
Comment (dirkmueller) [inline crowbar_framework/app/views/barclamp/cinder/_edit_attributes.html.haml]: no

File: chef/data_bags/crowbar/migrate/cinder/060_update_eternus.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
+def upgrade(ta, td, a, d)
+  ta_eternus = ta["volume_defaults"]["eternus"]
+
+  a["volumes"].each do |volume|
+    next if volume["backend_driver"] != "eternus"
+    volume["eternus"]["snappool"] = volume["eternus"]["pool"]
+    volume["eternus"]["pool"] = ta_eternus["eternus"]["pool"]
```
Comment (vuntz) [inline chef/data_bags/crowbar/migrate/cinder/060_update_eternus.rb]: Shouldn't we just keep the old value? The default is empty, which probably makes no sense.

File: chef/data_bags/crowbar/template-cinder.json [hunk 6]
```diff
@@ -131,7 +132,7 @@
     "cinder": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 47,
+      "schema-revision": 60,
```
Comment (toabctl) [inline chef/data_bags/crowbar/template-cinder.json]: bump to 100 iirc

**Comment by toabctl** [comment]
@dirkmueller can you address the comments, please?

**Comment by toabctl** [comment]
+1

**Comment by mjura** [comment]
+1

**Comment by aplanas** [comment]
@dirkmueller rebase?

**Pull request closed without merging**